packages/dm-core/src/components/MediaContent.tsx

```diff
@@ -102,7 +107,7 @@ export function MediaContent(props: MediaContentProps): ReactElement {
             <label className="meta-label">Filetype:</label>
             <span> {meta.filetype}</span>
             <label className="meta-label">Filesize:</label>
-            <span> {formatBytes(meta.fileSize)}</span>
+            {/*<span> {formatBytes(meta.fileSize)}</span>*/}
```

Comment:
Why remove this?

Reply:
oh, Thats not intended removed! thanks!

Reply:
Had to move that function over to `dm-core` though..